lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -107,14 +107,14 @@ class QueryBuilder
     /**
      * The index of the first result to retrieve.
      *
-     * @var int
+     * @var int|null
```

Comment:
It's not supposed to be nullable. The "non specified" first result is the same as `0`.

Reply:
Default value of property this is `null`, init value is not set in constructor, so it should be marked as nullable.

Reply:
The default value should be `0`, not `null` unless it breaks some existing API contract.